homeassistant/components/growatt_server/strings.json

```
@@ -0,0 +1,27 @@
+{
+    "config": {
+        "abort": {
+            "no_plants": "No plants have been found on this account"
+        },
+        "error": {
+            "auth_error": "Username or password is incorrect"
+        },
+        "step": {
+            "plant": {
+                "data": {
+                    "plant_id": "Plant"
+                },
+                "title": "Select your plant"
+            },
+            "user": {
+                "data": {
+                    "name": "Name",
+                    "password": "Password",
+                    "username": "Username"
```

```suggestion
                    "name": "[%key:common::config_flow::data::name%]",
                    "password": "[%key:common::config_flow::data::name%]",
                    "username": "[%key:common::config_flow::data::username%]"
```
Please use reference strings